cmd/monitor/collector/bootstrap-agent.yaml

```diff
@@ -7,6 +7,12 @@ health:
 
 erda.oap.collector.core:
   pipelines:
+    raws:
```

Comment:
maybe the name should be `raw_metrics`.

Reply:
https://github.com/erda-project/erda/blob/5068330f0cdecec54ea0f4647e7fc5dd4d5e2ee1/internal/tools/monitor/oap/collector/core/config/config.go#L41-L48

due to `PipelineWrap`, we only could use `raws` to implement this
TODO: read config from file